app/organization/contact-settings/route.js

```diff
@@ -2,11 +2,25 @@ import Ember from 'ember';
 import DS from 'ember-data';
 
 export default Ember.Route.extend({
+  model() {
+    const organization = this.modelFor('organization');
+    const billingDetail = this.store.find('billing-detail', organization.get('id'));
+    return Ember.RSVP.hash({
+      organization: organization,
+      billingDetail: billingDetail
+    });
+  },
+  
+  setupController(controller, model) {
+    controller.set('model', model.organization);
+    controller.set('billingDetail', model.billingDetail);
+  },
+
   afterModel(model) {
     return Ember.RSVP.all([
-      model.get('users'),
-      model.get('securityOfficer'),
-      model.get('billingContact')
+      model.organization.get('users'),
+      model.organization.get('securityOfficer'),
+      model.organization.get('billingContact')
```

Comment:
Shouldn't billing contact be pulled from `billingDetail` now, and not organization?